backend/app/views/spree/admin/shared/_address_form.html.erb

```diff
@@ -37,8 +37,9 @@
 
```

Comment:
Bug fix, If you load the address page trigger the JS to organise the Zipcode and States fields for the country.